frontend/src/components/Shared/DashboardSideNavbar/DashboardSideNavbar.jsx

```
@@ -0,0 +1,229 @@
+              {(isOpen || isHovering) && <span className="ml-3 text-sm">Logout</span>}
+            </button>
+          </div>
+        </div>
+      </nav>
+
+      {/* Mobile menu toggle button */}
+      {isMobile && (
+        <button
+          onClick={() => setIsOpen(!isOpen)}
+          className="fixed bottom-4 right-4 z-50 p-3 rounded-full bg-blue-600 text-white shadow-lg md:hidden"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)